ansible_ai_connect_chatbot/src/useChatbot/useChatbot.ts

```diff
@@ -369,13 +369,12 @@ export const useChatbot = () => {
             method: "POST",
             headers: {
               "Content-Type": "application/json",
+              Accept: "application/json,text/event-stream",
               "X-CSRFToken": csrfToken!,
             },
             body: JSON.stringify(chatRequest),
             async onopen(resp: any) {
-              if (resp.ok && resp.status === 200) {
-                setIsLoading(false);
```

Comment:
This line was used to remove the loading icon when the stream is opened. It is moved to line 397 when the first character is received.